src/go/options/configgenerator.go

```diff
@@ -93,10 +91,8 @@ func DefaultConfigGeneratorOptions() ConfigGeneratorOptions {
 	return ConfigGeneratorOptions{
 		CommonOptions:                 DefaultCommonOptions(),
 		BackendDnsLookupFamily:        "auto",
-		BackendProtocol:               "", // Required flag with no default
-		ClusterAddress:                "127.0.0.1",
+		BackendUri:                    "http://127.0.0.1:8082",
```

Comment:
BackendAddress? 

URI usually includes path. I think we dont have path part here.

Reply:
Done